### PR TITLE
allow to run unit-tests inside a container

### PR DESCRIPTION
**Pull request by SchSeba**

### CODE REVIEW

**Review by zeeke — APPROVED** [review]
LGTM

**Review by e0ne — APPROVED** [review]
LGTM

File: pkg/utils/testing.go [hunk 1]
```diff
@@ -85,6 +76,37 @@ func (fs *FakeFilesystem) Use() func() {
...
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			// use the test data if we run inside a container where we don't have the pci ids file
+			dir, err := os.Getwd()
+			if err != nil {
+				panic(fmt.Errorf("error getting working directory: %s", err.Error()))
+			}
+			//nolint: gocritic
+			pciIdsPath := filepath.Join(dir, "../../pkg/utils/testdata/pci.ids")
+			pciData, err = os.ReadFile(pciIdsPath)
```
Comment (adrianchiris) [inline pkg/utils/testing.go:94]: i think for tests we can always use this file. less variation when running test.

**Review by adrianchiris — COMMENTED** [review]
minor comment, otherwise LGTM

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**